overrides/partials/copyright.html

```diff
@@ -1,7 +1,7 @@
 <!-- Copyright and theme information -->
 <div class="md-copyright">
     <div class="md-copyright__highlight">
-      <div style="display: block;">The content on this website is for Flight Simulation use only. No warranty is provided; all content is offered “as is” without liability, and must not be used for real-world aviation or any other use. </div>
```

Comment:
Probably worth just adding CC by 4.0 in the footer 

```html
 VATPAC Standard Operating Procedures - Licensed under
      <a href="https://creativecommons.org/licenses/by/4.0/" rel="license noopener" target="_blank">
        CC BY 4.0
      </a>
      (except where otherwise noted).
```